Gems/Atom/RPI/Code/Source/RPI.Edit/Common/AssetUtils.cpp

```diff
@@ -46,6 +47,12 @@ namespace AZ
 
             AZStd::string ResolvePathReference(const AZStd::string& originatingSourceFilePath, const AZStd::string& referencedSourceFilePath)
             {
+                // Prevents "second join parameter is an absolute path" warnings in StringFunc::Path::Join below
+                if (AZ::IO::PathView{referencedSourceFilePath}.IsAbsolute())
+                {
+                    return referencedSourceFilePath;
+                }
+
```

Comment:
It would also be useful to return an empty string if referencedSourceFilePath is empty. I wasn't sure about changing this function so I wrapped it with another one that does the empty check on my local branch.